storybookjs/storybook
# react native Typescript transform

Issue:

## What I did
Added a flag to the react-native storybook cli to allow passing a custom config to the metro bundler

## How to test
Build the project and copied the compiled storybook-start.js, also created a sample project.

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
Yes, added

Does this need an update to the documentation?
Docs added

If your answer is yes to any of these, please make sure to include it in your PR.
